.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
 application_import_names = sagemaker_training, test, gethostname
 import-order-style = google
+ignore = E501, W503
```

Comment:
why we ignore E501 and W503 here? Is this just for compiler team's change or addressing generic issues?

Reply:
seems like there is a conflict between `black-format` and `flake8` for these errors. Only one of the rules can be satisfied. I chose to ignore these for flake8

Reply:
make sense